sweagent/frontend/src/components/controls/LRunControl.js

```diff
@@ -89,23 +92,6 @@ function LRunControl({
   }
 
   function getEnvInput() {
-    // Get environment configuration input controls based on the
```

Comment:
We will probably still want to bring this one back, so let's not remove it ;)

Reply:
Oops, restored in 3b7c3a2ef339f3a1e4cbf65bd1c55d93db4092c0 👍